database/news_schema.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const newsSchema = mongoose.Schema({
+  newsTitle: {type: String, required: true},
```

Comment:
it's the news file, so it should be clear enough.
anyway, if you feel ok with how it is now, it's just fine